Bottom: b20eedcb27496313e0a8d12d1871d6595007bf9d
Top:    dbcd73347cc0ce46dda61309f099631643a3e393
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-06 14:09:43 +0800

Refresh of virtio-net-basic-ebpf-offload

---

diff --git a/include/uapi/linux/virtio_net.h b/include/uapi/linux/virtio_net.h
index a3715a3..320cb42 100644
--- a/include/uapi/linux/virtio_net.h
+++ b/include/uapi/linux/virtio_net.h
@@ -261,4 +261,15 @@ struct virtio_net_ctrl_mq {
 #define VIRTIO_NET_CTRL_GUEST_OFFLOADS   5
 #define VIRTIO_NET_CTRL_GUEST_OFFLOADS_SET        0
 
+/* Control eBPF offloads
+ *
+ */
+
+struct virtio_net_ctrl_ebpf {
+       char insn[0];
+};
+
+#define VIRTIO_NET_CTRL_EBPF 6
+#define VIRTIO_NET_CTRL_EBPF_SET_OFFLOAD_PROG 0
+
 #endif /* _UAPI_LINUX_VIRTIO_NET_H */
